fe2-android/app/src/main/java/com/github/dedis/popstellar/utility/GeneralUtils.kt

```diff
@@ -146,4 +169,122 @@ object GeneralUtils {
       emptyArray()
     }
   }
+
+  /**
```

Comment:
I'm not very sure the code you put in GeneralUtils fits the purpose of the file, perhaps a dedicated file for this dynamic list setup could be better, or even creating a UI Utils file.

Reply:
I'll check it out thanks